docker/docker-compose.yml

```diff
@@ -22,7 +22,7 @@ services:
     extra_hosts:
       - "host.docker.internal:host-gateway"
   yorkie:
-    image: 'yorkieteam/yorkie:0.4.8'
+    image: 'yorkieteam/yorkie:0.4.10'
```

Comment:
@hackerwins `activateClientRequest` returns the following error for the server version 0.4.11 and above:
```
io.grpc.StatusException: UNAVAILABLE: upstream connect error or disconnect/reset before headers. reset reason: connection termination
```

Reply:
I've updated the protobuf models to the latest version([8454812](https://github.com/yorkie-team/yorkie-android-sdk/pull/147/commits/8454812a67390289dafc7061d44e94f4d98edae5)), but the issue still persists. 
It's working fine with the real server(`api.yorkie.dev`, 443), so I'll keep the CI local server version at 0.4.10 until the issue is resolved